rest/taskana-rest-spring/src/main/java/pro/taskana/task/rest/models/BulkOperationResultsRepresentationModel.java

```diff
@@ -0,0 +1,47 @@
+package pro.taskana.task.rest.models;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import org.springframework.hateoas.RepresentationModel;
+import pro.taskana.common.api.exceptions.ErrorCode;
+
+/** EntityModel class for {@link pro.taskana.common.api.BulkOperationResults}. */
+public class BulkOperationResultsRepresentationModel
+    extends RepresentationModel<BulkOperationResultsRepresentationModel> {
+
+  /** Map of keys to the stored information. */
+  protected Map<String, ErrorCode> tasksWithErrors = new HashMap<>();
+
+  public Map<String, ErrorCode> getTasksWithErrors() {
+    return tasksWithErrors;
+  }
+
+  public void setTasksWithErrors(Map<String, ErrorCode> tasksWithErrors) {
+    this.tasksWithErrors = tasksWithErrors;
+  }
+
+  protected boolean canEqual(Object other) {
```

Comment:
Why did you add this? We don't have it in any other RepresentationModel

Reply:
I have checked it again, it turns out this is actually not needed, so I will delete it